tests/sql_database/test_sql_database_pipeline.py

```diff
@@ -26,14 +30,81 @@ def test_load_sql_schema_loads_all_tables(sql_source_db: SQLAlchemySourceDB, des
     pipeline = dlt.pipeline(
         pipeline_name='sql_database',
         destination=destination_name,
-        dataset_name='my_sql_data',
+        dataset_name='test_sql_pipeline_' + uniq_id(),
         full_refresh=False
     )
-    load_info = pipeline.run(sql_database(sql_source_db.database_url, schema=sql_source_db.schema))
+    load_info = pipeline.run(sql_database(credentials=sql_source_db.credentials, schema=sql_source_db.schema))
     assert_load_info(load_info)
 
     with pipeline.sql_client() as c:
         for table, info in sql_source_db.table_infos.items():
             with c.execute_query(f"SELECT count(*) FROM {table}") as cur:
                 row = cur.fetchone()
                 assert row[0] == info['row_count']
+
+
+@pytest.mark.parametrize('destination_name', ALL_DESTINATIONS)
+def test_load_sql_table_names(sql_source_db: SQLAlchemySourceDB, destination_name: str) -> None:
+    pipeline = dlt.pipeline(
+        pipeline_name='sql_database',
+        destination=destination_name,
+        dataset_name='test_sql_pipeline_' + uniq_id(),
+        full_refresh=False
+    )
+    tables = ['chat_channel', 'chat_message']
+    load_info = pipeline.run(
+        sql_database(
+            credentials = sql_source_db.credentials,
+            schema=sql_source_db.schema,
+            table_names=tables
+        )
+    )
+    assert_load_info(load_info)
+
+    with pipeline.sql_client() as c:
+        for table in tables:
+            info = sql_source_db.table_infos[table]
+            with c.execute_query(f"SELECT count(*) FROM {table}") as cur:
+                row = cur.fetchone()
+                assert row[0] == info['row_count']
+
+
+@pytest.mark.parametrize('destination_name', ALL_DESTINATIONS)
+def test_load_sql_table_incremental(sql_source_db: SQLAlchemySourceDB, destination_name: str) -> None:
+    """Run pipeline twice. Insert more rows after first run
+    and ensure only those rows are stored after the second run.
+    """
+    os.environ['SOURCES__SQL_DATABASE__CHAT_MESSAGE__CURSOR_COLUMN'] = 'updated_at'
```

Comment:
it is really cool that it works! another thing you can do with dlt that never came to my mind :)